web/apps/labelstudio/src/app/RootPage.js

```diff
@@ -1,20 +1,22 @@
-import React from 'react';
-import { Menubar } from '../components/Menubar/Menubar';
-import { ProjectRoutes } from '../routes/ProjectRoutes';
+import React from "react";
+import { Menubar } from "../components/Menubar/Menubar";
+import { ProjectRoutes } from "../routes/ProjectRoutes";
 
-export const RootPage = ({content}) => {
-  const pinned = localStorage.getItem('sidebar-pinned') === 'true';
-  const opened = pinned && localStorage.getItem('sidebar-opened') === 'true';
+export const RootPage = ({ content }) => {
+  const pinned = localStorage.getItem("sidebar-pinned") === "true";
+  const opened = pinned && localStorage.getItem("sidebar-opened") === "true";
 
   return (
     <Menubar
       enabled={true}
       defaultOpened={opened}
       defaultPinned={pinned}
-      onSidebarToggle={(visible) => localStorage.setItem('sidebar-opened', visible)}
-      onSidebarPin={(pinned) => localStorage.setItem('sidebar-pinned', pinned)}
+      onSidebarToggle={(visible) =>
+        localStorage.setItem("sidebar-opened", visible)
+      }
+      onSidebarPin={(pinned) => localStorage.setItem("sidebar-pinned", pinned)}
```

Comment:
Can we increase the number of columns to more than 80? This will significantly reduce the size of this PR.